### ISSUE TITLE
Loss function "LogDepthL1Loss"

### ISSUE THREAD

**Issue by eecoder-dyf** [saic-vul/saic_depth_completion#6]
In "./saic_depth_completion/metrics/absolute.py", the code of class "LogDepthL1Loss" is：
```python
class LogDepthL1Loss(nn.Module):
    def __init__(self, eps=1e-5):
        super(LogDepthL1Loss, self).__init__()
        self.eps = eps
    def forward(self, pred, gt):
        mask = gt > self.eps
        diff = torch.abs(torch.log(gt[mask]) - pred[mask])
        return diff.mean()
```
When calculating difference bewteen ground truth and predicion, it was "torch.abs(torch.log(gt[mask]) - pred[mask]", or write in formula |lg(gt) - pred|, the correct form might be |lg(gt) -lg(pred)|?
